core/shared/src/test/scala/scalaz/zio/BaseCrossPlatformSpec.scala

```diff
@@ -0,0 +1,49 @@
+package scalaz.zio
+
+import java.util.{ Timer, TimerTask }
+
+import org.specs2.Specification
+import org.specs2.execute.AsResult
+import org.specs2.specification.core.{ AsExecution, Execution }
+import scalaz.zio.internal.PlatformLive
+import scalaz.zio.clock.Clock
+import scalaz.zio.console.Console
+import scalaz.zio.random.Random
+import scalaz.zio.system.System
+
+import scala.concurrent.duration.{ Duration, _ }
+import scala.concurrent.{ ExecutionContext, Future }
+
+abstract class BaseCrossPlatformSpec extends Specification with DefaultRuntime {
+
+  override val Platform = PlatformLive.makeDefault().withReportFailure(_ => ())
+
+  implicit val ec: ExecutionContext = ExecutionContext.Implicits.global
+
+  val DefaultTimeout: Duration = 60.seconds
+  val timer                    = new Timer()
+
+  implicit def zioAsExecution[A: AsResult, R >: Clock with Console with System with Random, E]
```

Comment:
This can be `R >: Environment`, which will include `Blocking` only for the JVM module.

Reply:
I'll apply this in some subsequent work - I had problems with build recently.